data/virt_autotest/host_unattended_installation_files/combustion/script.aarch64.ttyAMA0

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# combustion: network
+
+grub2_config() {
+    file=/etc/default/grub
+    sed -i 's/^\s*GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=15/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL=.*$/GRUB_TERMINAL="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_INPUT=.*$/GRUB_TERMINAL_INPUT="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_OUTPUT=.*$/GRUB_TERMINAL_OUTPUT="gfxterm console"/g' $file
+    sed -i 's/console=tty0//g' $file
+    sed -i 's/console=tty[^ ]*/console=ttyAMA0,115200/g' $file
+    echo "DEBUG: now $file is,"
+    cat $file
+    grub2-mkconfig -o /boot/grub2/grub.cfg
+}
+
+# Redirect output to the console
+exec > >(exec tee -a /dev/ttyAMA0) 2>&1
+
+# Configure grub2 in the system
+grub2_config
+
+# Keyboard
+systemd-firstboot --force --keymap=us
+
+# Registration
+if ! which SUSEConnect > /dev/null 2>&1; then
+    zypper --non-interactive install suseconnect-ng
```

Comment:
Why is this needed? Is there an image without suseconnect?

Reply:
I am not sure. I think it would be better to add this.

Reply:
If the system is not already registered, then the zypper command should fail anyway.

Reply:
I will take this into consideration and take action if necessary.

Reply:
I confirmed that even installing without repo returns successfully although no package will be installed. But I think you are right, so I removed this part.